Packs/CyberArk/Integrations/CyberArk/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## [Unreleased]
+
+
+## [19.11.1] - 2019-11-26
+These notes will be published in the next release notes
```

Comment:
this file is not needed anymore. if demisto-sdk init creates it - please open an issue.